samples/Calling/src/app/views/DisplayNameField.tsx

```diff
@@ -19,13 +19,13 @@ const TEXTFIELD_PLACEHOLDER = 'Enter a name';
 const TEXTFIELD_EMPTY_ERROR_MSG = 'Name cannot be empty';
 const TEXTFIELD_EXCEEDS_MAX_CHARS = `Name cannot exceed ${DISPLAY_NAME_MAX_CHARS} characters`;
 
+const hasValidLength = (name: string): boolean => {
+  return name.length <= DISPLAY_NAME_MAX_CHARS;
+};
+
 export const DisplayNameField = (props: DisplayNameFieldProps): JSX.Element => {
   const { setName, setEmptyWarning, isEmpty, defaultName } = props;
```

Comment:
Your original code looks like it should've worked (although this is better as it doesn't assign to isInvalidLength to true by default)

It looks like we never pass in setEmptyWarning and isEmpty: https://github.com/Azure/communication-ui-library/blob/cd5058489810b02b9eaf66d742381cd4cf36dc66/samples/Calling/src/app/views/HomeScreen.tsx#L86

So do we ever show the empty warning?

Reply:
Nope, we just disable the buttons.
But it can be passed through the props so I let that code sit there.